src/@utils/ddo.ts

```diff
@@ -1,15 +1,27 @@
 import { Asset, DDO, Service } from '@oceanprotocol/lib'
+interface Consumable {
+  status: number
+  message: string
+  result: boolean
+}
 
 export function getServiceByName(
   ddo: Asset | DDO,
-  name: 'access' | 'compute'
+  name: 'access' | 'compute' | 'metadata'
 ): Service {
   if (!ddo) return
 
   const service = ddo.services.filter((service) => service.type === name)[0]
   return service
 }
 
+export function getServiceById(ddo: Asset | DDO, serviceId: string): Service {
```

Comment:
This is a genuine question for basically everyone. Do we need one liner functions like this? Do they bring any benefit?

Reply:
from my perspective they have some benefits (small ones thats true 😃 ), like across the stack if you want to get a certain service by Id or name like in this particular example and you remember having this helpers and just call this method. Also the code readability  is better this way in my opinion if we have it like this 
`const service =  getSerivceById(asset, serviceId)
  doSomethingWithTheService(service)`  
rather than this 
`const service =  ddo.services.find((s) => s.id === serviceId)
  doSomethingWithTheService(service)`
  and maybe less codeclimate issues 😄 .
But totally got your point we can also get rid of them or just keep the ones we think bring some value and benefits to the stack 😃.

Reply:
fair enough